### ISSUE TITLE
Failed to obtain a valid EGL display

### ISSUE THREAD

**Issue by codelav** [klokantech/tileserver-gl#235]
Hello
I've installed tileserver-gl v2.2.0 on Ubuntu server 16.04
Downloaded map from openmaptiles.com
Try: 
`tileserver-gl map_20170703.mbtiles`
and get error libEGL.so.1 not found
checked packages for this file:
https://packages.ubuntu.com/search?suite=xenial&arch=amd64&searchon=contents&keywords=libegl.so.1

Installed: libegl1-mesa, later nvidia-361, same result:

> terminate called after throwing an instance of 'std::runtime_error'
>   what():  Failed to obtain a valid EGL display.
> 
> Aborted (core dumped)

Any ideas how to fix it?

**Comment by zhali** [comment]
same issue with you
   terminate called after throwing an instance of 'std::runtime_error'
   what():  Failed to obtain a valid EGL display.
   Aborted (core dumped)
do you solve it now?

**Comment by codelav** [comment]
@zhali, no, no time to solve it. I've switched to the version in the Docker. Works fine.

**Comment by zhali** [comment]
@codelav thank you , i should continue solve it...
